index.html

```diff
@@ -1091,7 +1091,7 @@ <h2 id="arrays">Array Functions</h2>
       <p id="sortedIndex">
         <b class="header">sortedIndex</b><code>_.sortedIndex(array, value, [iteratee], [context])</code>
         <br />
-        Uses a binary search to determine the index at which the <b>value</b>
+        Uses a binary search to determine the smallest index at which the <b>value</b>
```

Comment:
How about "lowest"?

Reply:
Please see my second commit, I also improved the test case for this scenario which was already present in your tests & I noticed that you already use word "smallest" in your code, for better consistency across the project let's leave "smallest" as is, although "lowest" wold do as well, of course.